migrations/20230704145138-create_stories_table.sql

```diff
@@ -0,0 +1,11 @@
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS stories (
+    story_id SERIAL PRIMARY KEY,
+    user_id INT,
+    story_content TEXT,
+    FOREIGN KEY (user_id) REFERENCES users(user_id),
+    UNIQUE (story_id)
+);
+
+-- +migrate Down
```

Comment:
Let us add the migrate down (rollback) section to reverse these changes, for good practice.

Also missing empty line at the end of the file.